lib/main.ts

```diff
@@ -139,16 +139,24 @@ export class Transpiler {
     defaultLibFileName = this.normalizeSlashes(defaultLibFileName);
     return {
       getSourceFile: (sourceName, languageVersion) => {
-        var path = sourceName;
+        var sourcePath = sourceName;
         if (sourceName === defaultLibFileName) {
-          path = ts.getDefaultLibFilePath(COMPILER_OPTIONS);
+          sourcePath = ts.getDefaultLibFilePath(COMPILER_OPTIONS);
+        }
+        let contents: string = null;
+        if (fs.existsSync(sourcePath)) {
+          contents = fs.readFileSync(sourcePath, 'UTF-8');
+        } else {
+          let nodePath = path.resolve('node_modules', sourceName);
+          if (!fs.existsSync(sourcePath)) return undefined;
+          contents = fs.readFileSync(nodePath, 'utf-8');
         }
-        if (!fs.existsSync(path)) return undefined;
-        var contents = fs.readFileSync(path, 'UTF-8');
         return ts.createSourceFile(sourceName, contents, COMPILER_OPTIONS.target, true);
       },
       writeFile(name, text, writeByteOrderMark) { fs.writeFile(name, text); },
-      fileExists: (filename) => fs.existsSync(filename),
+      fileExists: (filename) => {
+        return !!filename.match(/^rxjs\/.*\.d\.ts$/) || fs.existsSync(filename);
```

Comment:
Change to: `return !!filename.match(/^(\/)*rxjs\/.*\.d\.ts$/) || fs.existsSync(filename);`

This is because the path is '/rxjs/Observable', not 'rxjs/Observable'.